config/custom-environment-variables.json

```diff
@@ -1,4 +1,9 @@
 {
+  "appName": "PACKAGES_NAME",
+  "project": "PACKAGES_PROJECT",
+  "environment": "NODE_ENV",
+  "version": "PACKAGES_VERSION",
+  "deployment_env": "DEPLOYMENT_ENV",
```

Comment:
these shouldn't be needed, they are legacy vars from tactical
```suggestion
```

Reply:
Ok, removed